documentation/modules/managing/con-custom-resources-info.adoc

```diff
@@ -0,0 +1,126 @@
+The _short name_ for `Kafka` is `k`, so you can run `kubectl get k` to list all your deployed Kafka clusters.
+
+[source,shell]
+----
+kubectl get k
+
+NAME         DESIRED KAFKA REPLICAS   DESIRED ZK REPLICAS
+my-cluster   3                        3
+----
+
+.Long and short names for each Strimzi resource
+[cols="3*",options="header",stripes="none"]
+|===
+
+m|Strimzi resource      |Long name          |Short name
+
+| Kafka                 | kafka             | k
+| Kafka Topic           | kafkatopic        | kt
+| Kafka User            | kafkauser         | ku
+| Kafka Connect         | kafkaconnect      | kc
+| Kafka Connect S2I     | kafkaconnects2i   | kcs2i
+| Kafka Connector       | kafkaconnector    | kctr
+| Kafka Mirror Maker    | kafkamirrormaker  | kmm
+| Kafka Mirror Maker 2  | kafkamirrormaker2 | kmm2
+| Kafka Bridge          | kafkabridge       | kb
+| Kafka Rebalance       | kafkarebalance    | kr
+
+|===
+
+== Resource categories
+
+Categories of custom resources can also be used in `kubectl` commands.
+
+All Strimzi custom resources belong to the category `strimzi`, so you can use `strimzi` to get all the Strimzi resources with one command.
+
+For example running `kubectl get strimzi` lists all Strimzi custom resources in a given namespace.
+
+[source,shell]
+----
+kubectl get strimzi
+
+NAME                                   DESIRED KAFKA REPLICAS DESIRED ZK REPLICAS
+kafka.kafka.strimzi.io/my-cluster      3                      3
+
+NAME                                   PARTITIONS REPLICATION FACTOR
+kafkatopic.kafka.strimzi.io/kafka-apps 3          3
+
+NAME                                   AUTHENTICATION AUTHORIZATION
+kafkauser.kafka.strimzi.io/my-user     tls            simple
+----
+
+The `kubectl get strimzi -o name` command returns all resource types and resource names.
+The `-o name` option fetches the output in the _type/name_ format
+
+[source,shell]
+----
+kubectl get strimzi -o name
+
+kafka.kafka.strimzi.io/my-cluster
+kafkatopic.kafka.strimzi.io/kafka-apps
+kafkauser.kafka.strimzi.io/my-user
+----
+
+You can combine this `strimzi` command with others commands.
+For example, you can pass it into a `kubectl delete` command to delete all resources in a single command.
+
+[source,shell]
+----
+kubectl delete $(kubectl get strimzi -o name)
+
+kafka.kafka.strimzi.io "my-cluster" deleted
+kafkatopic.kafka.strimzi.io "kafka-apps" deleted
+kafkauser.kafka.strimzi.io "my-user" deleted
+----
+
+Deleting all resources in a single operation might be useful, for example,
+when your testing new Strimzi features.
+
+== Querying the status of sub-resources
+
+There are other values you can pass to the `-o` option.
+For example, by using `-o yaml` you get the output in YAML format.
+Usng `-o json` will return it as JSON.
+
+You can see all the options in `kubectl get --help`.
+
+One of the most useful options is `jsonpath`, which allows you to pass JSONPath expressions to query the Kubernetes API.
```

Comment:
What about adding this link on JSONPath? 
https://kubernetes.io/docs/reference/kubectl/jsonpath/

Reply:
Added. we have an attribute for the link